com.provar.plugins.provardx/src/commands/provar/metadatacache.ts

```diff
@@ -58,14 +59,34 @@ export default class metadatacache extends SfdxCommand {
         return {};
     }
 
-    this.ux.log("Metadata level" + ' : ' + metadataLevel);
+    this.ux.log("Metadata level" + ' : ' + caching);
     this.ux.log("Cache Path" + ' : ' + cachePath);
     this.ux.log("Property File" + ' : ' + propertyFile);
     this.ux.log("JSON" + ' : ' + json);
     this.ux.log("Log level" + ' : ' + logLevel);
+    
+    let properties = this.updatePropertiesWithOverrides(provarDxUtils.getProperties(), caching, cachePath, propertyFile);
+    let rawProperties = JSON.stringify(properties);
+
+    if (properties.cachePath == null) {
+      this.ux.error('Metadata Cache path is not specified');
+      return {};
+    }
+
+    let updateProperties = provarDxUtils.prepareRawProperties(rawProperties);
 
-    //TODO: Actual logic for metadata loading.
+    let jarPath = properties.provarHome +'/provardx/provardx.jar';
```

Comment:
add double quotes to path, it will cause issue if there is space in path.

Reply:
@himanshusharma1  Already added where we are using jarPath.